rhel/vex/parser_test.go

```diff
@@ -307,10 +307,10 @@ func TestParse(t *testing.T) {
 		expectedDeleted int
 	}{
 		{
-			name:            "six_advisories_two_deletions",
+			name:            "six_advisories_four_deletions",
 			filename:        "testdata/example_vex.jsonl",
 			expectedVulns:   546,
-			expectedDeleted: 2,
+			expectedDeleted: 4,
```

Comment:
guessing there are two vulns in there which do not have `fixed` nor `known_affected`?

Reply:
Yeah, there are a couple of `samba` advisories in the `example_vex.jsonl`